.gitmodules

```diff
@@ -81,7 +81,7 @@
 [submodule "mbed-os"]
 	path = third_party/mbed-os/repo
 	url = https://github.com/ARMmbed/mbed-os.git
-	branch = feature-chip
+	branch = master
```

Comment:
I don't think we can use it until https://github.com/ARMmbed/mbed-os/pull/14847 has been merged

Reply:
It should work with the code we have now but it won't with what's present upstream.

Reply:
Sure, I think we can merge it to the current development but check it again before upstreaming.